internal/vm/ioproxy.go

```diff
@@ -129,7 +130,12 @@ func (connectorPair *IOConnectorPair) proxy(
 
 		_, err := io.CopyBuffer(writer, reader, make([]byte, internal.DefaultBufferSize))
 		if err != nil {
-			logger.WithError(err).Error("error copying io")
+			if strings.Contains(err.Error(), "use of closed network connection") ||
+				strings.Contains(err.Error(), "file already closed") {
+				logger.Infof("connection was closed: %s", err)
```

Comment:
nit: just noticed this, it's slightly better to use `%v` for errors since `error` is technically an interface. Otherwise LGTM, thanks!

Reply:
Done